package/chalicelib/awsAuditor.py

```diff
@@ -15,7 +15,7 @@ def main():
 
     users = ['user1@email.com', 'user2@email.com']
 
-    r = ReportGenerator(start_date=start, end_date=end)
+    r = ReportGenerator(start_date=start, end_date=end, secret_name="your arn")
```

Comment:
We should also have the ARN pulled from the bucket too...

Reply:
Changed it to get the secret name from the bucket. Could you please add this to the json you uploaded? "email_credentials_name": ARN or friendly name

Reply:
There needs to be a corresponding change in awsAuditor. Lets talk tomorrow about how to implement this. I have an idea.